devtools/src/bin/inspect/main.rs

```diff
@@ -90,38 +95,72 @@ where
     })
 }
 
-fn inspect_bytes(bytes: Vec<u8>, context: Option<Context>) {
+fn inspect_bytes(bytes: Vec<u8>, context: Option<Context>, lookup: bool) {
     if let Some(block) = complete(&bytes, |r| block::Block::read(r)) {
         block::inspect(&block, context);
     } else if let Some(header) = complete(&bytes, |r| BlockHeader::read(r)) {
         block::inspect_header(&header, context);
     } else if let Some(tx) = complete(&bytes, |r| Transaction::read(r, BranchId::Nu5)) {
         // TODO: Take the branch ID used above from the context if present.
         // https://github.com/zcash/zcash/issues/6831
-        transaction::inspect(tx, context);
+        transaction::inspect(tx, context, None);
     } else {
         // It's not a known variable-length format. check fixed-length data formats.
-        inspect_fixed_length_bytes(bytes);
+        match bytes.len() {
+            32 => inspect_possible_hash(bytes.try_into().unwrap(), context, lookup),
+            64 => {
+                // Could be a signature
+                eprintln!("This is most likely a signature.");
```

Comment:
or a wtxid?

Reply:
Do we expose those with hex encodings anywhere?